### DISCUSSION TITLE
PhotonVision / AprilTags Integration

### DISCUSSION THREAD

**Discussion by jkleiber** [team401/2024-Robot-Code#138] [Ideas]
@charlesmackenzie one thing to try if the camera is still reporting an incorrect pose would be to set the camera's transform to the origin and at 0 angle and see if that does anything. You could also work with one camera enabled at a time to make sure the latest pose estimate is actually valid (for example: could we be getting an invalid pose estimate listed as the latest one from one of the back cameras or something?

**Comment by charlesmackenzie** [comment]
As of yesterday, the cameras report reasonable poses.
Our biggest problem now is that the Beelink turns itself off whenever we drive the robot. We were too busy practicing to actually catch when it stopped sending data, but I can pull logs off in the pit tonight. Electrical is looking at ways to give it a more stable power supply, but there's a very real chance we'll have to constantly re-localize during matches.
Apologies for not updating you yesterday-- it was pretty busy.

**Reply by jkleiber** [reply]
Thanks for the update. If the BeeLink is on regulated power it may be more of a connector issue (or we're drawing too much current in the rest of the system somehow)

Relocalizing will be fine for week 1, thanks for getting that supported
